### PR TITLE
[tests] Add ActionCreator 1rst tests

### PR DESCRIPTION
**Pull request by misstick**
##### 1. Add backbone for:
 - [x] `routerActionCreator`,
 - [x] `accountActionCreator`,
 - [x] `messageActionCreator`,

##### 2. Add tests for:
 - [x] `messageActionCreator.deleteMessage()` from a message.

##### 3. Spy for:
 - [x] success callback,
 - [x] error callback.

### CODE REVIEW

File: client/test/message_action_creator.spec.js [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+        // 1rst one for REQUEST
+        // Last one for SUCCESS
+        assert.equal(spyDispatcher.callCount, 2);
+
+        let args = spyDispatcher.getCall(1).args[0];
+        assert.equal(args.type, type);
+
+        let res = { conversationID: 'plop', text: 'coucou', html: 'coucou'};
+        assert.equal(args.value.action, action);
+        _.each(JSON.parse(args.value.message), (value, key) => {
```
Comment (m4dz) [inline client/test/message_action_creator.spec.js]: Shouldn't a [`deepEqual`](http://chaijs.com/api/assert/#method_deepequal) better in this case?

**Comment by m4dz** [comment]
Great!

I ask myself about creating a SpecSuperagent instead of just mocking requests with Sinon?

**Comment by misstick** [comment]
wouldnt it be less readable?

File: client/test/message_action_creator.spec.js (hunk 1 above)
Comment (misstick) [inline client/test/message_action_creator.spec.js]: In this case, we cant to compare:
 `let res = { "conversationID": "plop", "text": "coucou", "html": "coucou"};`with `let res = { conversationID: 'plop', text: 'coucou', html: 'coucou'};` and `deepEqual` doesnt resolve this issuer

Comment (m4dz) [inline client/test/message_action_creator.spec.js]: Seen from here, those 2 are strictly equals and `deepEqual` shoudl do the trick, nope?

**Comment by m4dz** [comment]
I hope not, and this is the gopal opf the mocking feature. But nwm, we can stuck at this one for this and we'll try with mocking later, al'right?

**Comment by misstick** [comment]
I'm OK with it.

File: client/test/message_action_creator.spec.js (hunk 1 above)
Comment (misstick) [inline client/test/message_action_creator.spec.js]: nope :( I tried...

**Pull request merged**